### PR TITLE
authoscope 0.8.0

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by carlocab** [comment]
```
Error: An exception occurred within a child process:
  Errno::ENOENT: No such file or directory - docs/badtouch.1
```
From
https://github.com/Homebrew/homebrew-core/blob/774e812525c4ff5a306f697fe37749da6c3bbbf7/Formula/badtouch.rb#L26

Looks like we now need `sphinx-doc` to make the manpage. We can maybe skip it.

File: Formula/authoscope.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
...
   desc "Scriptable network authentication cracker"
-  homepage "https://github.com/kpcyrd/badtouch"
-  url "https://github.com/kpcyrd/badtouch/archive/v0.7.3.tar.gz"
-  sha256 "cb09ae83bbe37e17ce1d5e86ba1c9f8b8dd1dc1b30e3f7d69d7cfd8db7ae5547"
+  homepage "https://github.com/kpcyrd/authoscope"
+  url "https://github.com/kpcyrd/authoscope/archive/v0.8.0.tar.gz"
+  sha256 "977df6f08a2fece7076f362bc9db6686b829de93ed3c29806d7b841a50bd9d1c"
   license "GPL-3.0-or-later"
 
   bottle do
```
Comment (carlocab) [inline Formula/authoscope.rb:8]:
```
authoscope:
  * New formulae in homebrew/core should not have a `bottle do` block
Error: 1 problem in 1 formula detected
```

**Comment by carlocab** [comment]
```
==> /opt/homebrew/Cellar/authoscope/0.8.0/bin/authoscope oneshot -vvx /private/tmp/authoscope-test-20210526-72897-g7g1q3/true.lua foo
error: Found argument 'oneshot' which wasn't expected, or isn't valid in this context

USAGE:
    authoscope [FLAGS] [OPTIONS] <SUBCOMMAND>

For more information try --help
```

File: Formula/authoscope.rb (hunk 1 above)
Comment (chenrui333) [reply Formula/authoscope.rb:8]: yes!

**Comment by chenrui333** [comment]
need to spend some time to get tests working.

**Comment by kpcyrd** [comment]
hey! here's my updated formula: https://github.com/kpcyrd/homebrew-core/commit/35632edf8c049a21a1b7ad013ea0b7613f818b26

I was about to file a PR myself, then noticed you've been faster :heart:  :)

**Review by carlocab — APPROVED** [review]

**Pull request closed without merging**